### DISCUSSION TITLE
Increase volume and normalize

### DISCUSSION THREAD

**Discussion by Protarios** [advplyr/audiobookshelf#1557] [Ideas]
As audiobooks often are recorded with a very volume, it would be great to have an increase volume button or even better an option to normalize volume on-the-fly.

On some audiobooks I have to put the volume to maximum, but when any other sound like a video or notification then is played this is way too load for headphones.
Sometimes the volume limitation even does not allow proper listening in load surroundings.

The issue is, at least German voiced audiobooks are often recorded with very low volume. Other audiobook players therefore allow to boost the audiobook volume in the app.

**Comment by advplyr** [comment]
There is a request open for this you can follow https://github.com/advplyr/audiobookshelf-app/issues/195
